pkg/scheduler/plugins/predicates/predicates.go

```diff
@@ -319,6 +320,11 @@ func (pp *predicatesPlugin) OnSessionOpen(ssn *framework.Session) {
 	// 7. VolumeZone
 	plugin, _ = volumezone.New(nil, handle)
 	volumeZoneFilter := plugin.(*volumezone.VolumeZone)
+	// 8. PodTopologySpread
+	// TODO: make this configurable? ref: https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints/#internal-default-constraints
```

Comment:
it's ok not support cluster level default constraint in this pr.

Reply:
done for that.